java/src/jmri/jmrit/logixng/tools/swing/LogixNGSwingBundle.properties

```diff
@@ -147,7 +147,7 @@ DeleteWithChildrenPrompt    = Are you sure you want to delete {0} and its childr
 VetoDeleteBean              = {0} {1} Can not be deleted\n{2}
 
 PopupMenuRenameSocket   = Rename socket
-PopupMenuAdd        = Add
+PopupMenuAdd        = Add  (Ctrl+N)
```

Comment:
The Ctrl+N is not needed for Add.  The popup menu does not need a shortcut for `add` since it is invoked using the mouse and the flow into the sub menus is a natural motion.

Reply:
Ctrl+N is useful if you prefer to use only the keyboard, not the mouse. With Ctrl+N, you can open the `New` menu and then move around with the cursor keys and select a new item.

This is also valuable for users with low vision who use screen readers. We have users that uses DecoderPro with screen readers. Today, the support for screen readers when developing LogixNGs is propably insufficient, but this is a step to supporting it.

Reply:
The popup menu does not support the N accelerator key since Add is not a JMenuItem.  Showing the `(Ctrl+N)` text is confusing.

Reply:
I have removed it now from the popup menu. Ctrl+N still works, but isn't shown in the menu.